src/test/java/com/booster/server/VocabularyEntryControllerTest.java

```diff
@@ -65,25 +73,61 @@ void returnVocabularyEntryListOfPredefinedSize() throws Exception {
+                .setLastSeenAt(LocalDateTime.of(LocalDate.EPOCH, LocalTime.MIN)));
 
         mockMvc.perform(MockMvcRequestBuilders.get("/vocabulary-entry/list?size=%s".formatted(expectedSize)))
                 .andDo(print())
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("$.length()").value(expectedSize))
-                .andExpect((jsonPath("$.[0].name").value(coalesce)))
-                .andExpect((jsonPath("$.[0].description").value(coalesceDescription)))
-                .andExpect((jsonPath("$.[1].name").value(robust)))
-                .andExpect((jsonPath("$.[1].description").value(robustDescription)));
-
+                .andExpect((jsonPath("$.[0].name").value(robust)))
+                .andExpect((jsonPath("$.[0].description").value(robustDescription)))
+                .andExpect((jsonPath("$.[1].name").value(coalesce)))
+                .andExpect((jsonPath("$.[1].description").value(coalesceDescription)));
 
         assertThat(vocabularyEntryRepository.findAll())
                 .hasSize(expectedSize)
                 .extracting("name", "description")
                 .containsExactly(Tuple.tuple(coalesce, coalesceDescription), Tuple.tuple(robust, robustDescription));
     }
 
+    @Test
+    void correctUpdateOfLastSeenAtPropertyOfVocabularyEntry() throws Exception {
+        assertThat(vocabularyEntryRepository.findAll()).isEmpty();
+
+        Long id1 = vocabularyEntryRepository.save(new VocabularyEntryEntity()
+                        .setName(coalesce)
+                        .setDescription(coalesceDescription))
+                .getId();
+        Long id2 = vocabularyEntryRepository.save(new VocabularyEntryEntity()
+                        .setName(robust)
+                        .setDescription(robustDescription))
+                .getId();
+        Long id3 = vocabularyEntryRepository.save(new VocabularyEntryEntity()
+                        .setName(contribution)
+                        .setDescription(contributionDescription))
+                .getId();
+
+        LocalDateTime lastSeenAt = LocalDateTime.now();
+        List<Long> ids = List.of(id1, id2);
+        mockMvc.perform(MockMvcRequestBuilders.patch("/vocabulary-entry")
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .content(objectMapper.writeValueAsString(new UpdateLastSeenAtInput()
+                                .setIds(ids)
+                                .setLastSeenAt(lastSeenAt))))
+                .andDo(print())
+                .andExpect(status().isOk());
+
+        assertTrue(vocabularyEntryRepository.findAllById(ids)
+                .stream()
+                .allMatch(n -> n.getLastSeenAt().equals(lastSeenAt)));
+        assertThat(vocabularyEntryRepository.findById(id3)
+                .map(VocabularyEntryEntity::getLastSeenAt).get())
+                .isEqualTo(LocalDateTime.of(LocalDate.EPOCH, LocalTime.MIN));
```

Comment:
We can get rid of the `get` method on `Optional` if we use the `contains` instead of `isEqualTo`.